EXAMPLE.md

```diff
@@ -0,0 +1,65 @@
+Let's make install spy and generate something we can compile with C and run.
+
+Requires:
+- Emscripten
+- Zig
```

Comment:
this is not really a prerequisite, since we are installing `ziglang` through `requirements.txt`.
The only annoying thing is that the `zig` executable is not in the `PATH` by default, see my comment here: https://github.com/spylang/spy/issues/70#issuecomment-2407473537

Reply:
Also added the solution